src/renderer/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.3.1"
 authors = ["Eyal Kalderon <ebkalderon@gmail.com>"]
 description = "High-level rendering engine with multiple backends"
 keywords = ["game", "engine", "renderer", "3d", "amethyst"]
+categories = ["rendering"]
```

Comment:
I thought that "rendering engine" is synonymous with "renderer"? Wouldn't `rendering::api` be better suited to `gfx-rs`, given that it deals with the Direct3D, OpenGL, Vulkan, and Metal API's?